source/extensions/filters/http/alpn/alpn_test.cc

```diff
@@ -167,6 +167,35 @@ TEST_F(AlpnFilterTest, EmptyOverrideAlpn) {
   }
 }
 
+TEST_F(AlpnFilterTest, AlpnOverrideFalse) {
+  NiceMock<StreamInfo::MockStreamInfo> stream_info;
+  NiceMock<StreamInfo::MockUpstreamInfo> upstream_info;
+  Envoy::Upstream::HostDescriptionConstSharedPtr upstream_host; // TODO: generate mock?, remove pointer
+  Envoy::Upstream::MetadataConstSharedPtr host_metadata; // TODO: generate mock?, remove pointer
```

Comment:
You don't have to generate mocks. You can apply this commit to fix this test: https://github.com/istio/proxy/pull/4847/commits/895c7c5f19449259eeb3f7afb90f468ed043ee9d

Reply:
You can see in [this PR](https://github.com/istio/proxy/pull/4847) that tests pass with this commit.